package.json

```diff
@@ -119,6 +119,8 @@
     "isomorphic-fetch": "^2.2.1",
     "jest": "^24.3.0",
     "jsdom": "^12.2.0",
+    "lighthouse-plugin-publisher-ads": "./node_modules/lighthouse-plugin-publisher-ads-wrapper/lighthouse-plugin-publisher-ads/",
+    "lighthouse-plugin-publisher-ads-wrapper": "https://github.com/googleads/publisher-ads-lighthouse-plugin",
```

Comment:
it would be really helpful if `lighthouse-plugin-publisher-ads` was published on npm so we didn't have to add the wrapper from github and then add the plugin with a relative path into the wrapper module :)

Reply:
`yarn` errors for me:

```
error Package "lighthouse-plugin-publisher-ads" refers to a non-existing file '"/usr/local/google/home/cjamcl/code/lighthouse/node_modules/lighthouse-plugin-publisher-ads-wrapper/lighthouse-plugin-publisher-ads"'.

```

I assume this only worked for you because you initially had `lighthouse-plugin-publisher-ads-wrapper` installed before adding the other one.

Reply:
![image](https://user-images.githubusercontent.com/2301202/68255405-bbd55680-fff2-11e9-8ce8-777d897f4688.png)

Reply:
> yarn errors for me

Yeah, we'll get it published before landing. Too bad the hack isn't reproducible from package.json though.